src/app/features/orders/components/order-card/order-card.component.ts

```diff
@@ -84,7 +81,7 @@ export class OrderCardComponent {
 
   public duration() {
     if (!this.segments().length) return '';
-    return calculateStopDuration(this.segments()[this.segments().length - 1].time[1], this.segments()[0].time[0]);
+    return calculateTrainStopDuration(this.segments()[this.segments().length - 1].time[1], this.segments()[0].time[0]);
```

Comment:
sample variable name: `departureDate/Time`
```suggestion
    const departureDate = this.segments()[0].time[0];
    const arrivalDate = this.segments()[this.segments().length - 1].time[1];
    return calculateTrainStopDuration(arrivalDate, departureDate);
```